webpack/dev-server.js

```diff
@@ -31,16 +31,7 @@ Object.keys(config.entry).forEach(entryName => {
 
 config.plugins = [
   new webpack.HotModuleReplacementPlugin(),
-  new ReactRefreshWebpackPlugin({
-    overlay: {
-      entry: '@pmmmwh/react-refresh-webpack-plugin/client/ErrorOverlayEntry',
-      module: '@pmmmwh/react-refresh-webpack-plugin/overlay',
-      sockIntegration: 'wds',
```

Comment:
my understanding is that these values are important for HMR outside of the overlay itself

Reply:
Only just go back around to testing this. It works fine, all these config values came from. It's not described in the docs.